docs/plugins/plugin-development-guide/implementation.rst

```diff
@@ -0,0 +1,217 @@
+Implementation
+--------------
+
+The goal of our plugin is simple - we need to extend ``ProductVariant`` entity and provide new flag, that could be set
+on product variant form. Following customizations use information provided in **Sylius Customization Guide**, especially
+from :doc:`customizing models</customization/model>`, :doc:`form</customization/form>` and :doc:`template</customization/template>`
+pages.
+
+.. attention::
+
+    ``PluginSkeleton`` is focused on delivering the most friendly and testable environment. That's why in ``tests/Application`` directory,
+    there is a **tiny Sylius application** placed, with your plugin already used. Thanks to that, you can test your plugin with Behat scenarios
+    **within** Sylius application without installing it to any test app manually! There is, however, one important consequence of such an architecture.
```

Comment:
why bold on `within`? and `Everything` below? :D

Reply:
To note about an awesome feature (you can test plugin in the application without installing it into the application 🎉) and make reader 100% aware about the need of copying everything that is needed to the test application :)